pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt

```diff
@@ -138,43 +140,56 @@ val orienteringOmRettigheterOgPlikterAlder =
         includePhrase(VedleggPlikterHvorforMeldeAP_001)
         showIf(
             barnetilleggVedvirk_innvilgetBarnetillegFellesbarn
-                    and barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn
-                    and not(ektefelletilleggVedvirk_innvilgetEktefelletillegg)
+                and barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn
+                and not(ektefelletilleggVedvirk_innvilgetEktefelletillegg)
         ) {
-            includePhrase(VedleggPlikterRettTilBarnetilleggAP_001)
+            includePhrase(
+                VedleggPlikterRettTilBarnetilleggAP_001(
+                    harTilleggForFlereBarn = harTilleggForFlereBarn
```

Comment:
No need for the first part of harTilleggForFlereBarn = harTilleggForFlereBarn. You can also put this includephrase on one line.